spec/game_runner_spec.rb

```diff
@@ -0,0 +1,42 @@
+require_relative 'spec_helper'
+require_relative '../lib/game_runner'
+
+RSpec.describe GameRunner, "#run" do
+  def game
```

Comment:
You don't need all these methods. You can do them inline inside the test.

Reply:
✅